a10_neutron_lbaas/tests/unit/v1/test_handler_hm.py

```diff
@@ -44,31 +44,38 @@ def fake_hm(self, type):
 
     def test_create_ping(self):
         self.a.hm.create(None, self.fake_hm('PING'), 'p01')
+        time.sleep(5)
```

Comment:
umm, no.  no, no, hell no, and no.  sleep to make something appear to run is never ever ever ever right.  ever.